src/tests/components/SearchForm.test.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import SearchForm from "../../components/SearchForm";
+
+describe("SearchForm", () => {
+  const validProps = {
+    searchText: "Manchester",
+    setSearchText: () => {},
+    onSubmit: () => {},
+  };
+
+  it("renders correctly", () => {
```

Comment:
Snapshot is failing here, should make sure these are up to date whenever you run your tests!